src/transformStateWithClones.js

```diff
@@ -6,8 +6,33 @@
  *
  * @return {Object[]}
  */
-function transformStateWithClones(state, actions) {
-  // write code here
+function cloneObject(obj) {
+  // Create a shallow clone of the object
+  return { ...obj };
 }
 
+function transformStateWithClones(state, actions) {
+  const stateHistory = [cloneObject(state)];
```

Comment:
```suggestion
  const stateHistory = [];
```
and we can remove the slice at all